src/platforms/flutter/upload-debug.mdx

```diff
@@ -26,6 +26,8 @@ Errors raised from the native layer in Flutter apps require certain debug inform
 
 Sentry requires a dSYM upload to symbolicate your crash logs. The symbolication process unscrambles Apple’s crash logs to reveal the function, file names, and line numbers of the crash. [Learn how to upload the dSYM files](/platforms/apple/dsym/).
 
+If you use the `split-debug-info` and `obfuscate` flags, you need to upload the `*.dSYM` files instead of the `*.symbols` files generated by the Flutter build, the `*.dSYM` files are located in the `build` folder of your project and not the given `split-debug-info` folder.
```

Comment:
```suggestion
If you use the `split-debug-info` and `obfuscate` flags, you need to upload the `*.dSYM` files instead of the `*.symbols` files generated by the Flutter build. The `*.dSYM` files are located in the `build` folder of your project and not the given `split-debug-info` folder.
```